spec/features/links/links_spec.rb

```diff
@@ -21,9 +21,9 @@
       expect(page).to have_table_row('4', 'Interaction 2 No link', '', 'Add link')
     end
 
-    it "shows 'No link' when editing a blank link" do
+    it "shows an empty cell when editing a blank link" do
       within('.table') { click_on('Add link', match: :first) }
-      expect(page).to have_field('link_url', with: 'No link')
+      expect(page.find_by_id('link_url').value).to be_blank
```

Comment:
It's always a shame to move away from the capybara helpers. Does `expect(page).to have_field('link_url', with: '')` not work?